Add Term Entry - HTML <center>

SSwiniarski opened the pull request:
<!--
👋 Hi, thanks for sending a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

Add term entry for the HTML center tag. Prompted by the target keyword list.

### Type of Change

<!--- Please delete or cross off the bullet point(s) that are irrelevant to this PR: -->

- Adding a new entry

### Checklist

<!-- Please check ALL the boxes: -->

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

Dusch4593 commented on content/html/concepts/tags/terms/center/center.md:
```diff
@@ -0,0 +1,37 @@
+---
+Title: '<center>'
+Description: 'Depreciated block-level element that displays its contents centered horizontally in the containing element. CSS should be used instead.'
```
```suggestion
Description: 'Displays its contents centered horizontally in the containing element. This is now deprecated and CSS should be used instead.'
```

Dusch4593 commented on content/html/concepts/tags/terms/center/center.md:
````diff
@@ -0,0 +1,37 @@
+
+Historical use:
+
+```html
````
```suggestion
```html
<!-- Don't use this! -->
```

Dusch4593 commented on content/html/concepts/tags/terms/center/center.md:
````diff
@@ -0,0 +1,37 @@
+
+Modern alternative:
+
+```html
````
```suggestion
```html
<!-- Use this, instead. :) -->
```

Dusch4593 commented on content/html/concepts/tags/terms/center/center.md:
```diff
@@ -0,0 +1,37 @@
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<center>` tag was used historically to center elements, usually text, within their containing element. It has been removed from the HTML specification, and while some browsers might support it for backwards compatibility, it should no longer be used.
```
```suggestion
The `<center>` tag was used historically to center content, usually text, within their containing elements. It has since been removed and deprecated from the HTML specification, and while some browsers might support it for backwards compatibility, it should no longer be used.
```

Dusch4593 reviewed (changes requested):
@SSwiniarski Left a few comments/suggestions for you to look over, but this entry is pretty solid overall! 👍

SSwiniarski commented:
@Dusch4593, I incorporated your few suggestions.

KTom101 commented on content/html/concepts/tags/terms/center/center.md:
```diff
@@ -0,0 +1,39 @@
+The `<center>` tag was used historically to center content, usually text, within its containing elements. It has since been deprecated and removed from the HTML specification. While some browsers might support it for backwards compatibility, it should no longer be used.
+
+The current practice in HTML is to use the [CSS](https://www.codecademy.com/resources/docs/css) property [`text-align`](https://www.codecademy.com/resources/docs/css/typography/text-align) to center text and to use properties of the [CSS box model](https://www.codecademy.com/resources/docs/css/box-model) to center other elements.
+
```
Add syntax section here. And perhaps mention it's usually found within the `<body>` container tag. I think that would be helpful for those who may be trying to look for it in older documents.

KTom101 reviewed (changes requested):
@SSwiniarski Nice piece. I only have one change.

SSwiniarski commented:
@KTom101, I implemented your suggestion.

@Dusch4593, @yangc95 FYI- I added a tag, "obsolete" that would be appropriate for historical entries like this.

Dusch4593 commented:
@SSwiniarski That's super fair. Would you mind adding that tag to the `documentation/tags.md` file and pushing to this PR?

SSwiniarski commented:
@Dusch4593, I added it to a sperate PR.

SSwiniarski commented:
@KTom101, sign off on this and we can merge.

Dusch4593 commented on content/html/concepts/tags/terms/center/center.md:
````diff
@@ -0,0 +1,48 @@
+</center>
+```
+
+Modern alternative:
````
```suggestion
The modern alternative is using the [`<div>`](https://www.codecademy.com/resources/docs/html/tags/div) tag, instead:
```

Dusch4593 commented:
Hey @SSwiniarski had one more suggestion before we merged. For the suggested alternative, it would be good to link to the relevant entry for that tag (in this case, the `<div>` tag). 😄

SSwiniarski commented:
@Dusch4593, applied your suggestion.

Dusch4593 commented:
@SSwiniarski @KTom101 

Sweet! PR is merged and once the entry goes live, we can view it here: https://www.codecademy.com/resources/docs/html/tags/center
